source/Grid/Grid.js

```diff
@@ -381,6 +384,31 @@ export default class Grid extends PureComponent {
     })
   }
 
+  /**
+   * Ensure offset position is visible
+   * Useful for animating position changes
+   */
+  scrollToPosition ({
+    scrollLeft = 0,
+    scrollTop = 0
+  } = {}) {
+    this._setScrollPosition({ scrollLeft, scrollTop })
+  }
+
+  /**
+   * Gets a calculated value to be used for `scrollLeft`
+   */
+  getScrollLeft (props = this.props, state = this.state) {
```

Comment:
What's the reason for these 2 methods to accept `props` and `state` as parameters instead of just using the current values (`this.props` and `this.state`)? External components really shouldn't know or care about or touch `instance.state`. And passing in `props` seems like it would make the method more complicated to use.

I was imagining something more like `getScrollLeft(columnIndex: number): number`- or maybe even `getOffsetForCell({ columnIndex, rowIndex }) => { scrollLeft, scrollTop }`.

Were you trying to allow this method to be called during `componentWillReceiveProps` / `componentWillUpdate` with a newer version of props than `Grid` currently had? I'm wary of that- in part because of the disconnect between externally-visible `props` and internally managed `state`.

**Edit for clarity**: It's okay for methods like `_getCalculatedScrollLeft` to accept params for `props` and `state` since they're private/internal methods. I just don't think we should expose this publicly.